engine/src/mac-dialog.mm

```diff
@@ -563,7 +563,7 @@ - (BOOL)panel:(id)sender shouldShowFilename:(NSString *)filename
 
 ////////////////////////////////////////////////////////////////////////////////
 
-void MCPlatformBeginFolderOrFileDialog(MCPlatformFileDialogKind p_kind, MCPlatformWindowRef p_owner, MCStringRef p_title, MCStringRef p_prompt, MCStringRef *p_types, uint4 p_type_count, MCStringRef p_initial, bool p_is_file)
+void MCPlatformBeginFolderOrFileDialog(MCPlatformFileDialogKind p_kind, MCPlatformWindowRef p_owner, MCStringRef p_title, MCStringRef p_prompt, MCStringRef *p_types, uint4 p_type_count, MCStringRef p_initial)
```

Comment:
For clarity, perhaps the order of params should be:

kind, owner, title, prompt, initial, types = nullptr, type_count = 0

Using default params, will make the use of this in the case of being a folder dialog easier to read.